src/js/components/DataChart/doc.js

```diff
@@ -87,8 +120,24 @@ export const doc = DataChart => {
       ]),
       PropTypes.string,
     ]).description(`The spacing between the axes and the Charts.`),
+    guide: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({
+        x: PropTypes.shape({
+          granularity: granularityType,
+        }),
+        y: PropTypes.shape({
+          granularity: granularityType,
+        }),
+      }),
+    ]).description('TBD'),
+    legend: PropTypes.bool.description('Whether to include a legend'),
```

Comment:
What about allowing customization of Legend from the theme?

Reply:
Is there a way for me to control the font size of the Legend?

Reply:
not yet. I was intending to get some mileage as is and then add more theming later.